alinka/widget/components.py

```diff
@@ -20,6 +47,15 @@ def __init__(self, text, parent, min_lenght: int | None = None):
         self.line_edit = QLineEdit(self)
         if min_lenght:
```

Comment:
`min_length` - there’s typo. Not part of this change but it should be fixed at some point. :)